chart/flux/README.md

```diff
@@ -197,6 +197,7 @@ The following tables lists the configurable parameters of the Weave Flux chart a
 | `token`                                           | `None`                                               | Weave Cloud service token
 | `extraEnvs`                                       | `[]`                                                 | Extra environment variables for the Flux pod(s)
 | `rbac.create`                                     | `true`                                               | If `true`, create and use RBAC resources
+| `clusterScope`                                    | `true`                                               | If `false`, will only create rbac resources for the namespace where flux is deployed to and will set k8s-whitelist-namespace arg in flux and allowNamespace arg in flux-operator to that namespace
```

Comment:
`--k8s-whitelist-namespace` is deprecated now, it's `--k8s-allow-namespace`

Reply:
cool, fixed this now